guide/payments/contacts.md

```diff
@@ -0,0 +1,229 @@
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
+### Adding a contact
+
+The most basic interaction is that a user manually adds a contact by entering their name and an address. This is likely not the most common user flow, as most contacts will be added to incoming payment requests, as illustrated further below. But wallets should generally support manual options, as users may not be able to avoid them.
+
+{% include image-gallery.html pages = page.imagesAddContact %}
+
+### Supporting various payment request formats
+
+Based on your use case, your application may choose to only support certain payment request formats. It should still recognize all formats, and provide the user with appropriate feedback. You may also choose to add warnings before permitting use of certain formats if there are privacy and security risks the user should be aware of.
+
+{% include image-gallery.html pages = page.imagesSupportedFormats %}
+
+### Contact editing
+
+The contacts screen should offer various features for editing and organization. Also see [Requesting bitcoin &middot; Contact cards]({{ '/guide/payments/request/#contact-cards' | relative_url }}).
+
+{% include image-gallery.html pages = page.imagesMultiAddressContact %}
+
+### Importing an address
+
+In this scenario, the user has copied a Lightning address to the clipboard and opened the app. The clipboard is automatically scanned, the app identifies the addres and offers the user appropriate options. Here, the user adds the address as a new contact.
+
+{% include image-gallery.html pages = page.imagesImportAddress %}
+
+This flow can also be initiated by scanning a QR code or tapping a payment link. For more, see [Sending bitcoin &middot; Payment entry points]({{ '/guide/payments/send/#payment-entry-points' | relative_url }}).
+
+### Importing an invoice
+
+Here, the user has scanned a payment request and assigns a contact to the payment.
+
+Some payment request formats may include an address that can receive payments repeatedly. In this case, the address is added to the contact for future use. Otherwise, only the payment is linked.
+
+Invoices may also contain recipient names. These can be used to suggest the name for a new contact to the user. Names can be spoofed, so they should not be automatically assigned without user approval.
+
+{% include image-gallery.html pages = page.imagesPayInvoice %}
+
+### Sending from the home screen
+
+Initiating a payment can be as simple as entering an amount and tapping a contact that has a re-usable address associated. Otherwise, an extra step is needed to manually enter a destination address.
+
+{% include image-gallery.html pages = page.imagesHome %}
+
+### Adding a contact to an outgoing invoice
+
+Contacts can also be assigned to invoices that will be paid by others.
+
+{% include image-gallery.html pages = page.imagesReceive %}
+
+### Adding a contact to a completed payment
```

Comment:
Seems to be an overlap with this section and the Importing an invoice section.

Reply:
Very different flows. One is when importing an invoice, making a payment an assigning the contact right in that flow. The other is is going back to completed payments and adding the contact.

Reply:
But you don't import an invoice, invoices are single use, you're still importing an address (as you've defined them).